template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-FSharp/Company.WebApplication1.fsproj

```diff
@@ -16,8 +16,8 @@
   <ItemGroup>
     <PackageReference Include="FSharp.Core" Version="4.1.*" />
     <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0-preview1-24507" />
-    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="2.0.0-preview1-24507" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0-preview1-24566" />
```

Comment:
Not sure if I should be updating FSharp projects as well but it seems to be using our metapackage.